save-backend/src/main/resources/application-prod.properties

```diff
@@ -1,3 +1,4 @@
 spring.datasource.url=jdbc:mysql://192.168.0.250:3306/save_cloud
 spring.quartz.job-store-type=jdbc
 spring.quartz.jdbc.initialize-schema=ALWAYS
+backend.loki-service-url=http://127.0.0.1:9110
```

Comment:
I believe it should also be `loki:9110`, because `127.0.0.1` is used only in configuration of loki-docker-driver, which is a plugin of Docker daemon

Reply:
thanks!